applications/FluidDynamicsApplication/custom_processes/compute_pressure_coefficient_process.h

```diff
@@ -0,0 +1,199 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Eduard Gómez
+//
+//
+
+#ifndef KRATOS_COMUTE_PRESSURE_COEFFICIENT_PROCESS_H
+#define KRATOS_COMUTE_PRESSURE_COEFFICIENT_PROCESS_H
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+#include "includes/define.h"
+#include "processes/process.h"
+#include "includes/kratos_parameters.h"
+
+// Application includes
+
+
+namespace Kratos
+{
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+using NodeType = ModelPart::NodeType;
```

Comment:
Here you're defining the `NodeType` for the entire Kratos namespace. The definition should go into the class.

Reply:
Good catch. Fixed.